src/mission_execution/src/MissionExecution.cpp

```diff
@@ -0,0 +1,59 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright 2018 Intel Corporation. All Rights Reserved.
+
+#include "mission_execution/MissionExecution.hpp"
+
+#include <stdio.h>
+
+MissionExecution::MissionExecution()
+: Node("mission_execution"),
+  missionPlan_(nullptr)
+{
+  RCLCPP_INFO(get_logger(), "MissionExecution::MissionExecution");
+
+  cmdSub_ = create_subscription<std_msgs::msg::String>("MissionExecutionCmd",
+      std::bind(&MissionExecution::onCmdReceived, this, std::placeholders::_1));
+}
+
+MissionExecution::~MissionExecution()
+{
+  RCLCPP_INFO(get_logger(), "MissionExecution::~MissionExecution");

```

Comment:
Could we make this a little more user friendly log information like "Mission Execution Node is shutting down" or something like that?